CMakeLists.txt

```diff
@@ -204,8 +204,8 @@ endif()
 
 # target to run cpplint.py for all configured sources
 set(LINT_TARGET lint CACHE STRING "Name of C++ and python style checker target")
```

Comment:
Why is this a configurable variable? What's the use case here?

Reply:
That's how it was done for the other linter/formater so I copied it but it's true that it does not make much sense. I removed it from the configurable variable as well as the others.